examples/tasks/smart-all-metrics-file.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@mkuculyma Sorry but I am not familiar enough with what needs to be updated regarding Godeps versions of components used by this plugin to be able to do this in the PR. 

Furthermore, we'd like this to be merged ASAP (with the other dynamic metric PR #25 ) because our official integration closing for the current delivery is by the end of tomorrow, deeply sorry about this.